### PR TITLE
Homework2 done!

### PR DESCRIPTION
**Pull request by naregharibyann**
HOMEWORK 2

### CODE REVIEW

File: src/main/java/Lesson2/Homework2.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package Lesson2;
+
+public class Homework2 {
+    public static void main(String[] args) {
+        String armCode = "ARM";
+        String usCode = "USA";
+        int armBudget = 500000;
+        int usBudget = 1000000;
```
Comment (karengrigoryanpicsart) [inline src/main/java/Lesson2/Homework2.java:8]: instead of variables, you could use array

File: src/main/java/Lesson2/Homework2.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+        if (countryCode == armCode) {
+            if (money >= armBudget) {
+                System.out.println("You can visit Armenia.");
+            } else {
+                System.out.println("You don't have enough money to visit Armenia.");
+            }
+        } else if (countryCode == usCode) {
+            if (money >= usBudget) {
+                System.out.println("You can visit USA");
```
Comment (karengrigoryanpicsart) [inline src/main/java/Lesson2/Homework2.java:21]: if the country is Armenia, we will get "You can visit USA" which is not correct

File: src/main/java/Lesson2/Homework2.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            } else {
+                System.out.println(" You don't have enough money to visit USA.");
```
Comment (karengrigoryanpicsart) [inline src/main/java/Lesson2/Homework2.java:23]: the same here

File: src/main/java/Lesson2/Homework2.java (hunk 1 above)
Comment (naregharibyann) [reply src/main/java/Lesson2/Homework2.java:8]: I totally agree, that was the reason of requesting learning materials

File: src/main/java/Lesson2/Homework2.java (hunk 2 above)
Comment (naregharibyann) [reply src/main/java/Lesson2/Homework2.java:21]: How?? second if condition is under else if, thus first statement should be true in order to reach to the second if part.

Can you please tell me which values to set to countryCode and money variables, in order to have this case.

File: src/main/java/Lesson2/Homework2.java (hunk 3 above)
Comment (naregharibyann) [reply src/main/java/Lesson2/Homework2.java:23]: Same here

File: src/main/java/Lesson2/Homework2.java (hunk 2 above)
Comment (karengrigoryanpicsart) [reply src/main/java/Lesson2/Homework2.java:21]: there is no check for Armenia

**Pull request merged**